cli/index.ts

```diff
@@ -511,7 +519,9 @@ const builtYargs = createYargsCli({
         };
 
         runner.onChange(printExecutedGraph);
-        printExecutedGraph(await runner.resultPromise());
+        const runResult = await runner.resultPromise()
+        printExecutedGraph(runResult);
+        return ((runResult.status === dataform.RunResult.ExecutionStatus.FAILED) ? 1 : 0);
```

Comment:
are you using Prettier with vscode? i'd expect it to remove all these brackets upon save (if you have format-on-save turned on)

Reply:
👍